pkg/ebpf/events/events.go

```diff
@@ -218,7 +218,7 @@ func capturePolicyEvents(ctx context.Context, ringbufferdata <-chan []byte, enab
 				if rb.Verdict == VerdictDeny {
 					dropCountTotal.WithLabelValues(direction).Add(float64(1))
 					dropBytesTotal.WithLabelValues(direction).Add(float64(rb.PacketSz))
-					log().Infof("Flow Info: Src IP: %s Src Port: %d Dest IP: %s Dest Port: %d Proto: %s Verdict: %s Direction: %s", utils.ConvByteToIPv6(rb.SourceIP).String(), rb.SourcePort,
+					log().Errorf("Flow Info: Src IP: %s Src Port: %d Dest IP: %s Dest Port: %d Proto: %s Verdict: %s Direction: %s", utils.ConvByteToIPv6(rb.SourceIP).String(), rb.SourcePort,
```

Comment:
but it is not error right? This will provide incorrect signal during troubleshooting.

Reply:
Ideally we should update this interface to support `V` method where we can optionally set log level.

Reply:
agreed. if we want to log DENY irrespective of log level, we should de-couple it from current log config which is being set from outside (would a new logger whose config is explicitly set in code work?)